### ISSUE TITLE
Switch input event PRESS_LONG_RELEASED for device Universalschalter is not supported

### ISSUE THREAD

**Issue by dolodobendan** [tschamm/boschshc-hass#106]
I have a new warning in my log:

```
This error originated from a custom integration.

Logger: custom_components.bosch_shc
Source: custom_components/bosch_shc/__init__.py:244
Integration: Bosch SHC (documentation, issues)
First occurred: 00:17:13 (1 occurrences)
Last logged: 00:17:13

Switch input event PRESS_LONG_RELEASED for device Universalschalter is not supported, please open issue
```

It asked me nicely to open an issue, so here we go.

**Comment by tschamm** [comment]
That’s a good point, the event was added to the API recently, but I did not add the corresponding code to create an event. Will do so im September when back from holiday.

**Comment by dolodobendan** [comment]
Thank you, and have a nice holiday!

**Comment by tschamm** [comment]
Fixed in 691fdbb
Please check whether this is working as expected

**Comment by dolodobendan** [comment]
Haven't seen anything in the logs, thank you.
